### DISCUSSION TITLE
Deployed site to cloudflare and UI is broken

### DISCUSSION THREAD

**Discussion by diwashrestha** [razonyang/hugo-theme-bootstrap#683] [Q&A] [answered]
I deployed a site to Netlify and Cloudflare. The site is deployed but the UI is not working. Its looks like just plain HTML.

site link: https://d-blog.pages.dev/

![Screenshot from 2022-08-29 08-37-38](https://user-images.githubusercontent.com/19636247/187113709-0bbc4192-2b36-4e4d-8ebd-a590c2e14654.png)

I am using https://github.com/razonyang/hugo-theme-bootstrap-skeleton. What may be the reason behind it?

**Comment by razonyang** [comment, marked as answer]
That's because you're loading assets from the skeleton demo site, you'll need to change the `baseURL` on production(`config/production/config.yaml`).
I think you missed the [deployment](https://github.com/razonyang/hugo-theme-bootstrap-skeleton#deployment) part.

**Reply by diwashrestha** [reply]
Thank you for your quick response.
Changes in baseURL and relativeURLs fixed the issue.
